docs/handlers.rst

```diff
@@ -114,3 +117,26 @@ If you want to have a default handler, executed if no message is previously capt
         # previously defined handlers matches the message
         handlers.default(default_response),
     ]
+
+
+Create a new handler
+--------------------
+
+The interface of a handler is quite simple. To create a new one, everything needed is a class with a method called ``match`` and an attribute called ``view``. The method ``match`` define if that handler should be used or not to respond to the new message. It must receive a ``Message`` as a parameter and return a boolean. The attribute ``view`` will be used if the ``match`` returns ``True``.
+
+In the example below, there is a handler that checks if message has more than 50 words:
+
+.. code-block:: python
+
+    class LongTextHandler:
```

Comment:
I think i would be clearer if the example had a pattern to compare, like `StartsWithHandler` or something similar. I got confused if this did the same thing.